AnnService/inc/Core/KDT/ParameterDefinitionList.h

```diff
@@ -12,7 +12,6 @@ DefineKDTParameter(m_sDeleteDataPointsFilename, std::string, std::string("delete
 DefineKDTParameter(m_pTrees.m_iTreeNumber, int, 1L, "KDTNumber")
 DefineKDTParameter(m_pTrees.m_numTopDimensionKDTSplit, int, 5L, "NumTopDimensionKDTSplit")
 DefineKDTParameter(m_pTrees.m_iSamples, int, 100L, "Samples")
-DefineKDTParameter(m_pTrees.m_bOldVersion, bool, false, "IsOldVersion")
```

Comment:
should not delete this parameter, otherwise the test will fail due to wrong version of KDT load

Reply:
Sorry, added it back in.  Is this parameter needed in `BKT/ParameterDefinitionList.h` as well?